### ISSUE TITLE
SVG version of the logo - not possible to resize current version

### ISSUE THREAD

**Issue by wd15** [materialsinnovation/pymks#317]
@davidbrough1, @ajmedford, do either of you have the original design for the logo -- it would be really nice to have an SVG version so it can be resized easily. For example, this doesn't look good right now,

https://github.com/wd15/pymks/blob/64d2826e1c2fec310d49d4360c45319a2c54a256/README.md

**Comment by ajmedford** [comment]
See attached. The colors are acting funny in Ubuntu for the .svg image, but the .pdf or .eps should also work.

[pymks_logo.zip](https://github.com/materialsinnovation/pymks/files/444985/pymks_logo.zip)

**Comment by wd15** [comment]
@ajmedford, thanks so much! The SVG looks great. 

BTW, here is the version of the documentation that I'm putting together on Readthedocs

http://pymks.readthedocs.io/en/issue145-docs/rst/README.html

The Markdown gets rendered to RST and I'm trying to get the notebooks automatically with nbsphinx. Will let you know when it's ready with a pull request. Basically, `pandoc` and `jupyter nbconvert` will no longer be required in the Makefile.

**Comment by wd15** [comment]
@ajmedford, the pymks_logo.svg file that you posted is not actually a real SVG, it's simply a binary embedded in SVG tags. It still can't be resized correctly. This isn't that important, it's just a logo, but if you have the original file that created the logo, i.e. the geometry specification then we can use that to create the SVG. The SVG can't be created from a binary format.

```
 <?xml version="1.0" encoding="UTF-8" standalone="no"?>
<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 20010904//EN"
  "http://www.w3.org/TR/2001/REC-SVG-20010904/DTD/svg10.dtd">
<svg width="497" height="473">
  <image id="image0" width="497" height="473" x="0" y="0"
    xlink:href="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAfEAAAHZEAIAAABJP0kyAAAAIGNIUk0AA
HomAACAhAAA+gAAAIDo
AAB1MAAA6mAAADqYAAAXcJy6UTwAAAAGYktHRP///////wlY99wAAAAJcEhZcwAAAEgAAABIAEbJ
az4AAAAHdElNRQfgCB4NFDEtjp5zAAA5XnpUWHRSYXcgcHJvZmlsZSB0eXBlIHhtcAAAeNrtvUly
```

**Comment by ajmedford** [comment]
Strange... I had some issues with colors between Mac/Ubuntu and something must have gotten lost when I was converting around with ImageMagic. The attached .svg should work - it is SVG 1.0, and while it isn't exactly human readable there doesn't seem to be any binary in there, and the colors look correct (at least on Mac). I included a .pdf as well for good measure.

[pymks_logo.zip](https://github.com/materialsinnovation/pymks/files/468455/pymks_logo.zip)

**Comment by wd15** [comment]
@ajmedford, the logo is good now, thanks so much.

**Comment by wd15** [comment]
This was fixed with #319
